### PR TITLE
Update README and repo cleanup

### PR DESCRIPTION
**Pull request by villaseca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+
+![front](front.png)
+
+![back](back.png)
+
+The motors are driven by two [DRV8830](www.ti.com/product/drv8830) I²C motor controllers, one for the left wheels and the other for the right wheels. The slave I²C address for the left wheels driver are ``0xC0h`` and ``0xC1h``, for write and read respectively. As for the right wheels driver, they are ``0xD0h`` and ``0xD1h``, respectively.
+
+Currently the maximum current for each motor is 500mA.
+
+The [ADC](http://www.ti.com/product/ADS1015) reads three reflective sensors at address ``0x48h``. The left, center and right sensors are read at analog inputs ``AIN0``, ``AIN1`` and ``AIN2`` respectively.
```
Comment (jbaldwinroberts) [inline README.md]: What are the reflective sensors? What are the left, right and center sensors? 
I think we need some more info here

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+
+It supports a 2-cell LiPo battery with an on-board charger which should be supplied from 9V to 12V, at 1A max.
```
Comment (jbaldwinroberts) [inline README.md]: How are we gonna charge this thing? What happens if I have a 2.5A 12V charger that I want to use? Maybe specify the connector needed.

**Review by jbaldwinroberts — COMMENTED** [review]
Do you need a review of the layout/schematics? I can clone and take a look in KiCAD if needed?

File: README.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+
+The only manufacturing constrain of this board is that it needs 2oz or higher copper weight.
+
+Besides the components listed on the BoM, the rest of the parts needed are:
+
+* 4 x Micro Metal Gearmotor ([example](https://shop.pimoroni.com/products/micro-metal-gearmotor-extended-back-shaft))
+* 4 x Micro Metal Gearmotor ([example](https://shop.pimoroni.com/products/pimoroni-gearmotor-bracket-pair))
```
Comment (lurch) [inline README.md]: You've got "4 x Micro Metal Gearmotor" listed twice? (albeit with different URLs)

**Pull request merged**